projects/kubernetes/autoscaler/README.md

```diff
@@ -0,0 +1,15 @@
+## **Kubernetes Cluster Autoscaler**
+![1.20 Version](https://img.shields.io/badge/1--20%20version-v1.20.1-blue)
+![1.21 Version](https://img.shields.io/badge/1--21%20version-v1.21.3-blue)
+![1.22 Version](https://img.shields.io/badge/1--22%20version-v1.22.6-blue)
+![1.23 Version](https://img.shields.io/badge/1--23%20version-v1.23.1-blue)
+![Build Status](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiYzQ3dzRvZHVqU2MvYnVuMzB3QmRZdVd1U1RabVorWnlqTXBYUGxDSGk2NXJXUU12c3pLQ25CQUdaQmlNUE84S0JIVVZUU0ozeTJJb3J0NWxNejNSbzk4PSIsIml2UGFyYW1ldGVyU3BlYyI6IkhLNTZwQ0hiZDZVUzVRdXYiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
+
+[Autoscaler](https://github.com/kubernetes/autoscaler) defines the cluster autoscaler.
+
+You can find the latest version of this image [on ECR Public Gallery](https://gallery.ecr.aws/eks-anywhere/kubernetes/autoscaler).
+
+### Updating
+
+The source of truth for golang build version is [here](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-go-version-should-be-used-to-compile-ca).
```

Comment:
Copy one of the other projects updating section and use as a template to fill this out some.